src/main/kotlin/io/sip3/twig/ce/controller/SearchController.kt

```diff
@@ -70,8 +70,8 @@ class SearchController {
         val query = request.query
                 .replace(SIP_METHOD_INVITE, Strings.EMPTY)
 
-        if (query.contains("sip.") && query.contains("rtp.")) {
-            throw UnsupportedOperationException("Complex search by `sip.` and `rtp.` filters is not supported.")
+        if (query.contains("sip.") && (query.contains("rtp.") || query.contains("rtcp."))) {
```

Comment:
Here at the moment we don't support search by any 2 of those 3. So condition should be modified.